notebooks/Example_LLH_setup_fixed_dirs.ipynb

```diff
@@ -787,76 +897,599 @@
+    "\n",
+    "---\n",
+    "\n",
+    "This function takes in arguments for theta and phi and updates the class variables, **.theta** and **.phi**, with the new values.\n",
+    "\n",
+    "These values are compared to **.resp_theta** and **.resp_phi**, which hold the values for the theta and phi values that were last used to query the response files. If the angle separation between (**.theta**, **.phi**) and (**.resp_theta**, **.resp_phi**) is larger than **._resp_update**, then the new theta and phi values are used to query new response files. \n",
+    "\n",
+    "If the angle separation is less than **._resp_update**, we must also check if the angle separation is larger than **._trans_theta**. If so, then the transmission values must also be updated as they are also queried by theta and phi values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#The theta and phi values prior to running set_theta_phi()\n",
+    "sig_mod.set_theta_phi(theta, phi)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### .set_flux_params(flux_params)\n",
+    "---\n",
+    "**.set_flux_params(flux_params)** takes a list of parameters, flux_params, as an argument and updates the class variable, **.flux_param**, with the new values. The following normed dpis are also set: **.normed_photon_fluxes**, **.normed_comp_flor_rate_dpis**, **.normed_photoe_rate_dpis**, **.normed_rate_dpis**, and lastly,  **.normed_err_rate_dpis**. The normed dpis are useful for varying parameters such as the amplitude, A, without modifying the underlying dpis.\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "### .get_rate_dpis(params)\n",
+    "---\n",
+    "The **.get_rate_dpis(params)** function returns the dpis corresponding to the dictionary of parameters given. These parameters contain values for theta, phi, and the normalization constant, A. \n",
+    "\n",
+    "For new angles, if the angle separation is larger than **_trans_update** , then .set_theta_phi() and .set_flux_params() are run again for the new parameters. Finally, the rate_dpis are calculated after multiplying **.normed_rate_dpis** by the normalization constant, A, and returned.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sig_mod"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## **Background Model**\n",
+    "---\n",
+    "\n",
+    "The background model encompasses any of the non-GRB sources that show up in the image. This can include contributions from the cosmic ray background or any number of sources that are known to be sources of gamma rays. The background model consists mainly of a diffuse model due to the cosmic ray background combined with point source models for each of the known sources within the field of view.\n",
```

Comment:
cosmic ray background --> cosmic X/gamma-ray background

"cosmic rays" are different (confusingly), and are not really modelled in this way.